PowerUp-2018/src/com/spikes2212/robot/Commands/commandGroups/MoveLiftToTarget.java

```diff
@@ -0,0 +1,36 @@
+package com.spikes2212.robot.Commands.commandGroups;
+
+import com.spikes2212.genericsubsystems.commands.MoveBasicSubsystem;
+import com.spikes2212.robot.Robot;
+import com.spikes2212.robot.SubsystemConstants;
+import com.spikes2212.robot.Commands.MoveLift;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class MoveLiftToTarget extends CommandGroup {
+
+	public MoveLiftToTarget(boolean up) {
+		addSequential(new MoveBasicSubsystem(Robot.liftLocker, SubsystemConstants.LiftLocker.CLOSE_SPEED));
```

Comment:
rename "up". maybe to "moveUp" or something. also add docs to understand what this variable means